### PR TITLE
[IMP] Add date/time of report generation in XLS export (issue #191)

### LINKED ISSUE #191: Add date/time of report generation in XLS export
This could be added at the bottom of the report.

### PR DESCRIPTION
**Pull request by apineux**
closes #191

### CODE REVIEW

**Comment by sbidoul** [comment]
@apineux is covered by the ACSONE ECLA.

File: mis_builder/report/mis_report_instance_xlsx.py [hunk 1]
```diff
@@ -142,6 +144,20 @@ def generate_xlsx_report(self, workbook, data, objects):
...
+        footer_format = workbook.add_format({
+            'italic': True, 'font_color': '#202020', 'size': 9})
+        now = datetime.now()
+        lang_model = self.env['res.lang']
+        lang = lang_model._lang_get(self.env.user.lang)
+        date_format = lang.date_format
+        time_format = lang.time_format
+        create_date = _("Generated on {} at {}").format(
+            now.strftime(date_format),
+            now.strftime(time_format))
```
Comment (sbidoul) [inline mis_builder/report/mis_report_instance_xlsx.py]: I think there something better to get now in the user timezone and language. Maybe `tools.format_tz()`?

**Comment by sbidoul** [comment]
Can you also add a file named `191.feature` in readme/newfragments. This project uses [towncrier](https://pypi.org/project/towncrier/) to generate the changelog.

File: mis_builder/report/mis_report_instance_xlsx.py (hunk 1 above)
Comment (apineux) [reply mis_builder/report/mis_report_instance_xlsx.py]: I'm not sure that we want to depend on a function declared in addons.mail.models.mail_template?
We can still use:
```suggestion
utc_now = datetime.now()
tz = pytz.timezone(self.env.user.tz or "UTC")
now = pytz.utc.localize(utc_now).astimezone(tz)
create_date = _("Generated on {} at {}").format(
    now.strftime(date_format),
    now.strftime(time_format))
```

Comment (sbidoul) [reply mis_builder/report/mis_report_instance_xlsx.py]: @apineux indeed depending on the mail module is not a good idea.
I think `fields.Datetime.context_timestamp` is the way to go.

**Comment by sbidoul** [comment]
The newfragment should contain the changelog entry as a short paragraph;

**Review by sbidoul — CHANGES_REQUESTED** [review]
Better way to get local time

**Comment by apineux** [comment]
@sbidoul Can you validate (or unvalidate) this development? Thanks

**Review by sbidoul — APPROVED** [review]

**Comment by sbidoul** [comment]
Squashed. Please /ocabot merge

**Pull request merged**